services/core-api/app/api/projects/project_summary/models/project_summary_authorization.py

```diff
@@ -3,6 +3,8 @@
 from sqlalchemy.dialects.postgresql import UUID
 from cerberus import Validator
 import json
+from flask import current_app
+from werkzeug.exceptions import BadRequest
```

Comment:
I'm seeing only imports added to this file, so they probably aren't being used and can be removed

Reply:
These are removed now